File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/LoadCbxLanguageInfo.java
package nta.med.data.model.ihis.bass;

import java.math.BigDecimal;
import java.sql.Timestamp;

public class LoadCbxLanguageInfo {
	
	private Integer propertyId;
	private String propertyCode;
	private String propertyName;
	private String propertyValue;
	private String propertyType;
	private String moduleType;
	private BigDecimal defaultFlg;
	private Integer sortNo;
	private String description;
	private String locale;
	private String sysId;
	private String updId;
	private Timestamp created;
	private Timestamp updated;
	private BigDecimal activeFlg;
	public LoadCbxLanguageInfo(Integer propertyId, String propertyCode, String propertyName, String propertyValue,
			String propertyType, String moduleType, BigDecimal defaultFlg, Integer sortNo, String description,
			String locale, String sysId, String updId, Timestamp created, Timestamp updated, BigDecimal activeFlg) {
		super();
		this.propertyId = propertyId;
		this.propertyCode = propertyCode;
		this.propertyName = propertyName;
		this.propertyValue = propertyValue;
		this.propertyType = propertyType;
		this.moduleType = moduleType;
		this.defaultFlg = defaultFlg;
		this.sortNo = sortNo;
		this.description = description;
		this.locale = locale;
		this.sysId = sysId;
		this.updId = updId;
		this.created = created;
		this.updated = updated;
		this.activeFlg = activeFlg;
	}
	public Integer getPropertyId() {
		return propertyId;
	}
	public void setPropertyId(Integer propertyId) {
		this.propertyId = propertyId;
	}
	public String getPropertyCode() {
		return propertyCode;
	}
	public void setPropertyCode(String propertyCode) {
		this.propertyCode = propertyCode;
	}
	public String getPropertyName() {
		return propertyName;
	}
	public void setPropertyName(String propertyName) {
		this.propertyName = propertyName;
	}
	public String getPropertyValue() {
		return propertyValue;
	}
	public void setPropertyValue(String propertyValue) {
		this.propertyValue = propertyValue;
	}
	public String getPropertyType() {
		return propertyType;
	}
	public void setPropertyType(String propertyType) {
		this.propertyType = propertyType;
	}
	public String getModuleType() {
		return moduleType;
	}
	public void setModuleType(String moduleType) {
		this.moduleType = moduleType;
	}
	public BigDecimal getDefaultFlg() {
		return defaultFlg;
	}
	public void setDefaultFlg(BigDecimal defaultFlg) {
		this.defaultFlg = defaultFlg;
	}
	public Integer getSortNo() {
		return sortNo;
	}
	public void setSortNo(Integer sortNo) {
		this.sortNo = sortNo;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getLocale() {
		return locale;
	}
	public void setLocale(String locale) {
		this.locale = locale;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	public String getUpdId() {
		return updId;
	}
	public void setUpdId(String updId) {
		this.updId = updId;
	}
	public Timestamp getCreated() {
		return created;
	}
	public void setCreated(Timestamp created) {
		this.created = created;
	}
	public Timestamp getUpdated() {
		return updated;
	}
	public void setUpdated(Timestamp updated) {
		this.updated = updated;
	}
	public BigDecimal getActiveFlg() {
		return activeFlg;
	}
	public void setActiveFlg(BigDecimal activeFlg) {
		this.activeFlg = activeFlg;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/LoadCbxLanguageInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    public class LoadCbxLanguageInfo
    {
        private String _propertyId;
        private String _propertyCode;
        private String _propertyName;
        private String _propertyValue;
        private String _propertyType;
        private String _moduleType;
        private String _defaultFlg;
        private String _sortNo;
        private String _description;
        private String _locale;
        private String _sysId;
        private String _updId;
        private String _created;
        private String _updated;
        private String _activeFlg;

        public String PropertyId
        {
            get { return this._propertyId; }
            set { this._propertyId = value; }
        }

        public String PropertyCode
        {
            get { return this._propertyCode; }
            set { this._propertyCode = value; }
        }

        public String PropertyName
        {
            get { return this._propertyName; }
            set { this._propertyName = value; }
        }

        public String PropertyValue
        {
            get { return this._propertyValue; }
            set { this._propertyValue = value; }
        }

        public String PropertyType
        {
            get { return this._propertyType; }
            set { this._propertyType = value; }
        }

        public String ModuleType
        {
            get { return this._moduleType; }
            set { this._moduleType = value; }
        }

        public String DefaultFlg
        {
            get { return this._defaultFlg; }
            set { this._defaultFlg = value; }
        }

        public String SortNo
        {
            get { return this._sortNo; }
            set { this._sortNo = value; }
        }

        public String Description
        {
            get { return this._description; }
            set { this._description = value; }
        }

        public String Locale
        {
            get { return this._locale; }
            set { this._locale = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String UpdId
        {
            get { return this._updId; }
            set { this._updId = value; }
        }

        public String Created
        {
            get { return this._created; }
            set { this._created = value; }
        }

        public String Updated
        {
            get { return this._updated; }
            set { this._updated = value; }
        }

        public String ActiveFlg
        {
            get { return this._activeFlg; }
            set { this._activeFlg = value; }
        }

        public LoadCbxLanguageInfo() { }

        public LoadCbxLanguageInfo(String propertyId, String propertyCode, String propertyName, String propertyValue, String propertyType, String moduleType, String defaultFlg, String sortNo, String description, String locale, String sysId, String updId, String created, String updated, String activeFlg)
        {
            this._propertyId = propertyId;
            this._propertyCode = propertyCode;
            this._propertyName = propertyName;
            this._propertyValue = propertyValue;
            this._propertyType = propertyType;
            this._moduleType = moduleType;
            this._defaultFlg = defaultFlg;
            this._sortNo = sortNo;
            this._description = description;
            this._locale = locale;
            this._sysId = sysId;
            this._updId = updId;
            this._created = created;
            this._updated = updated;
            this._activeFlg = activeFlg;
        }

    }
}